databuilder/rcpai/base_data_loader.py

```diff
@@ -0,0 +1,135 @@
+import os
+import uuid
+
+from elasticsearch import Elasticsearch
+from pyhocon import ConfigFactory
+
+from databuilder.extractor.neo4j_extractor import Neo4jExtractor
+from databuilder.extractor.neo4j_es_last_updated_extractor import Neo4jEsLastUpdatedExtractor
+from databuilder.extractor.neo4j_search_data_extractor import Neo4jSearchDataExtractor
+from databuilder.job.job import DefaultJob
+from databuilder.loader.file_system_elasticsearch_json_loader import FSElasticsearchJSONLoader
+from databuilder.loader.file_system_neo4j_csv_loader import FsNeo4jCSVLoader
+from databuilder.publisher.elasticsearch_publisher import ElasticsearchPublisher
+from databuilder.publisher.neo4j_csv_publisher import Neo4jCsvPublisher
+from databuilder.task.task import DefaultTask
+from databuilder.transformer.base_transformer import NoopTransformer
+from databuilder.rcpai.base_arg_parser import Neo4JConf
+
+
+class BaseDataLoader(object):
```

Comment:
@we-z note how all of the shared logic (`create_es_publisher_sample_job`, `create_last_updated_job`) now lives in this class, which is extended by YugabyteSQLLoader and MinioLoader.

Reply:
sounds good, more efficient